packages/client-core/src/admin/components/Group/CreateGroup.tsx

```diff
@@ -80,13 +70,24 @@ const CreateGroup = (props: Props) => {
         description: '',
         scopeTypes: []
       })
+      handleClose(false)
     }
-    handleClose(false)
+  }
+
+  const scopeData: ScopeData[] = []
+  adminScopeTypeState.scopeTypes.value.forEach((el) => {
```

Comment:
you can use `map` here to declare and initialise the scopeData at the same time.

avoid usage of forEach as much as possible it use callback returns in the backend and doesn't stop execution.